plugins/rapture/nexus-rapture-plugin/src/main/resources/static/rapture/NX/controller/Bookmarking.js

```diff
@@ -126,10 +126,7 @@ Ext.define('NX.controller.Bookmarking', {
   onNavigate: function (token) {
     var me = this;
 
-    if (token !== Ext.History.bookmark) {
```

Comment:
Yes, it’s unrelated.

Since onNavigate is triggered when the bookmark changes, can this statement ever be false?